computingservices/DocumentServices/services/dts/redactionsummary.py

```diff
@@ -42,23 +42,26 @@ def __packaggesummary(self, message, documentids, pageflags, programareas):
             print("\n_pageflags",_pageflags)
             summarydata = []
             docpageflags = documentpageflag().get_documentpageflag(message.ministryrequestid, redactionlayerid, ordereddocids)
+            print("\n docpageflags",docpageflags)
```

Comment:
print statement left in?

Reply:
It's needed for redaction summary debugging to see if there is any data issue.

Reply:
We are keeping for logging purposes in prod I believe.